hicommonwealth/commonwealth
# [ERC20 Launchpad] - Token Signature Signing step

## Link to Issue
Closes: https://github.com/hicommonwealth/commonwealth/issues/8707

## Description of Changes
Added signature signing step for launching a token with some changes to previous steps. Stepwise, it now works like this

1. First we get a token info draft from the user
2. This token info is then used to generate token specific community data, and community is created
3. Then we require the user to sign signatures, and on success, we update the community to utilize this token (and TODO: save this token data in our DB).

## "How We Fixed It"
N/A

## Test Plan
1. Enable `FLAG_TOKENIZED_COMMUNITY`
2. Visit `/createTokenCommunity`
3. Complete the token and community forms (steps 1 and 2) - verify they work
4. On the signature signing (step 3), verify you are able to launch token without any issues (Note: you won't get a success screen ATM)

## Deployment Plan
N/A

## Other Considerations
N/A
